### PR TITLE
Saving reduced data to file in SANS  workflow

### PR DESCRIPTION
**Pull request by wpotrzebowski**
This is needed primarily for https://github.com/dmsc-nightly/scipp-sasview because currently, the workflow checks out the sans_file_saving branch and not main.

### CODE REVIEW

**Comment by nvaytet** [comment]
@wpotrzebowski the build is failing, can you fix?

File: docs/instruments/loki/sans2d_reduction.ipynb [hunk 1]
```diff
@@ -35,7 +35,8 @@
     "import scipp as sc\n",
     "from ess import loki, sans\n",
     "from ess.logging import configure_workflow\n",
-    "import scippneutron as scn"
+    "import scippneutron as scn\n",
+    "import scippnexus as snx"
```
Comment (nvaytet) [inline docs/instruments/loki/sans2d_reduction.ipynb]: How about
`import scippnexus.v2 as snx`
?

**Comment by wpotrzebowski** [comment]
It seems to be failing again... I will fix it but need to sort out my local env first

**Comment by wpotrzebowski** [comment]
Thanks @nvaytet. I was about to push this fix. Hope it works now.

File: docs/instruments/loki/sans2d_reduction.ipynb [hunk 2]
```diff
@@ -486,6 +488,29 @@
...
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "c3329275-3fb3-420c-b09f-704e241befe4",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "result.coords['Q'] = sc.midpoints(result.coords['Q'])\n",
+    "\n",
+    "with snx.File('test.nxs', 'w') as f:\n",
+    "    f['sasentry'] = nxcansas.SASentry(title='Reduced data', run=12345)\n",
```
Comment (nvaytet) [inline docs/instruments/loki/sans2d_reduction.ipynb]: Is the run number required, and is this why you made it up?

Comment (wpotrzebowski) [reply docs/instruments/loki/sans2d_reduction.ipynb]: It seems to be a required parameter but maybe something more reasonable can be put in there (I am thinking something corresponding to run on SANS2D).

**Review by nvaytet — APPROVED** [review]

**Comment by SimonHeybrock** [comment]
What is the plan here regarding the move to `esssans`?

**Comment by nvaytet** [comment]
> What is the plan here regarding the move to esssans?

This is something Wojciech did a long time ago, before `sciline` existed. So we will just merge here and then migrate over to `esssans` later.

**Pull request merged**